test-harness

```diff
@@ -97,8 +97,12 @@ def split_arguments(args = []):
             links_args.append(args[i])
     return (links_args, rest_args)
 
-def evaluate(name, code, config_file, stdout='', stderr='', exit = '0', env = None, filemode='', args='', ignore = None):
+def evaluate(name, code, config_file, old_printer='', stdout='', stderr='', exit = '0', env = None, filemode='', args='', ignore = None):
```

Comment:
These changes are fine whilst you are developing, but they must be reverted before this PR can be merged.

Reply:
I reverted this